demo/src/stream-bench.ts

```diff
@@ -126,60 +128,72 @@ async function main() {
     let schemaStream = Cord.Content.fromSchemaAndContent(
       schema,
       content,
-      aliceDid.uri,
-    ) 
-    console.log('schemaStream',schemaStream)
-
+      bobDid.uri,
+      aliceDid.uri
+    )
+    const document = await Cord.Document.fromContent({
+      content: schemaStream,
+      authorization: Alice,
+      registry: registry.identifier,
+      signCallback: async ({ data }) => ({
+        signature: aliceKeys.assertionMethod.sign(data),
+        keyType: aliceKeys.assertionMethod.type,
+      }),
+    })
     process.stdout.write(
       '  🔖  Extrinsic creation took ' +
         moment.duration(moment().diff(startTxPrep)).as('seconds').toFixed(3) +
         's\r'
     )
+
     try {
-      let txStream = await createStream(
-        aliceDid.uri,
-        aliceKeys,
-        async ({ data }) => ({
-          signature: aliceKeys.assertionMethod.sign(data),
-          keyType: aliceKeys.assertionMethod.type,
-        }),
-        content
+      const { streamHash } = Cord.Stream.fromDocument(document)
+      const streamTx = api.tx.stream.create(
```

Comment:
do we need a 'await' here?

Reply:
I did console.log ('streamTx') and I got the output 'Submittable'. Hence, I thought 'await' is not needed.